NineChronicles.Headless.Executable/Configuration.cs

```diff
@@ -0,0 +1,213 @@
+using System;
+using System.IO;
+using NineChronicles.Headless.Properties;
+
+namespace NineChronicles.Headless.Executable
+{
+    public class Configuration
+    {
+        public class DevConfiguration
+        {
+            public int BlockInterval { get; set; } = 100;
+            public int ReorgInterval { get; set; } = 100;
+        }
+
+        public string? AppProtocolVersionString { get; set; }
+
+        public string[]? TrustedAppProtocolVersionSignerStrings { get; set; }
+
+        public string? GenesisBlockPath { get; set; }
+        public string? Host { get; set; }
+        public ushort? Port { get; set; }
+
+        public string? SwarmPrivateKeyString { get; set; }
+
+        public int Workers { get; set; } = 5;
+
+        // Storage
+        public string? StoreType { get; set; }
+
+        public string? StorePath { get; set; } =
+            Path.Combine(
+                new string[]
+                {
+                    Environment.GetFolderPath(Environment.SpecialFolder.UserProfile),
+                    "AppData", "Local", "planetarium", "9c-main-partition"
+                }
+            );
```

Comment:
It seems good 👍🏻 But if you intended to target the 9c-launcher's store path, how about branching the store path by OS platform like 9c-launcher does?

https://github.com/planetarium/9c-launcher/blob/377138073823f1f6efd9bdfc3466fe47aa9ee4df/src/config.ts#L161-L166 

It seems it can use like https://learn.microsoft.com/en-us/dotnet/api/system.operatingsystem.ismacos?view=net-6.0#system-operatingsystem-ismacos 🤔 

Of course, it officially supports Windows yet so it can be passed now 🤔

Reply:
This may help: https://developers.redhat.com/blog/2018/11/07/dotnet-special-folder-api-linux